src/DemoUI/src/Views/resources/css/index.css

```diff
@@ -0,0 +1,1865 @@
+.pl-main-wrapper {
+    position: relative;
+    clear: both;
+    padding: 0 !important;
+    margin: -20px 0 0;
+    min-height: 500px;
+    border: 1px solid #dfdfdf;
+}
+
+.pl-header-wrapper {
+    height: 80px;
+    background-color: white;
+    padding: 0 !important;
+    border-bottom: 1px solid #dcdcdc;
+}
+
+.pl-header-row {
+    margin: 0 !important;
+}
+
+.pl-logo-wrapper {
+    width: 100%;
+    padding: 20px;
+    box-sizing: border-box;
+}
+
+.pl-dashboard-logo {
+    width: 100%;
+}
+
+.pl-page-wrapper {
+    display: flex;
+    height: 100%;
+}
+
+.pl-sidebar-wrapper {
+    position: relative;
+    background-color: #f5f5f5;
+    border-right: 1px solid #dcdcdc;
+    max-width: 250px;
+    width: 18%;
+    height: 100%;
+}
+
+.pl-content-wrapper {
+    height: 100%;
+    background-color: white;
+    padding-top: 20px;
+    flex-grow: 1;
+    position: relative;
+    box-sizing: border-box;
+}
+
+.pl-content-wrapper .row {
+    margin: 0 !important;
+}
+
+.pl-content-wrapper > .row {
+    height: 100% !important;
+    box-sizing: border-box;
+}
+
+.pl-content-wrapper-panel {
+    padding: 0 10px 0 20px;
+    height: 100%;
+    position: relative;
+    box-sizing: border-box;
+}
+
+.pl-content-wrapper-panel > .row {
+    height: 100%;
+}
+
+.pl-sidebar-link-wrapper {
+    text-align: center;
+    min-height: 50px;
+    padding: 0 !important;
+    cursor: pointer;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/packlink-dev/ecommerce_module_core/pullRequest?prid=5826400)